CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 59.3.1
```

Comment:
Updates to the version number should happen in a separate PR. For now, the changes should be listed as `# Unreleased`

Reply:
Oops - good to know! Will do that.